emails/models.py

```diff
@@ -120,11 +124,13 @@ def display_name(self):
         return self.fxa.extra_data.get('displayName')
 
     @property
-    def has_unlimited(self):
+    def has_premium(self):
         # FIXME: as we don't have all the tiers defined we are over-defining
         # this to mark the user as a premium user as well
         if not self.fxa:
             return False
+        if self.user.email.endswith('@mozilla.com'):
+            return True
```

Comment:
Changed this property to `has_premium` and returning `True` if their FXA ends with `@mozilla.com` so we can fox-food premium features.